parsl/executors/workqueue/executor.py

```diff
@@ -375,9 +414,10 @@ def __init__(self,
         self.shared_files = set()
         self.registered_files = set()
         self.worker_output = see_worker_output
-        self.full = True
+        self.full = False
```

Comment:
is this a fix for a bug or is it a behaviour change?

Reply:
I apologize, I missed this line when reading through the tidy-up changes. But what this does is enables full Work Queue monitoring. It was changed to false because CCTools monitoring was a bottleneck for our throughput tests, and removing it helped get higher throughput values for the tests. Perhaps, the monitoring could be added as an option for the executor? It is currently only enabled/disabled manually through the script right now.

Reply:
My gut feeling for new stuff in parsl is to default to lots of debugging when the code is new and then as time passes and we care more about throughput rather than debugging, reduce that level.

Other places in parsl have user controllable worker debugging - for example, see the `worker_debug` parameter to `HighThroughputExecutor` in parsl/executors/high_throughput/executor.py

Reply:
Ok, it sounds like it is best to set it as a parameter for the Work Queue Executor, but have it default to enable debugging.